### DISCUSSION TITLE
SofaContextAPI errors in SofaAPAPI-Unity3D

### DISCUSSION THREAD

**Discussion by PiccioFra** [sofa-framework/sofa#4169] [Build / Config / Environment] [answered]
Hi everyone, I'm having problems with SofaAPAPI-Unity3D, the asset that allows you to run SOFA simulation in background while your Unity3D project is running. I have followed the instructions at https://infinytech3d.com/getting-started/ but, although it manages to load the .unity examples, when i play the simulation it does nothing because i have problem with the SofaContextAPI. It gives me errors like: 
"SofaContextAPI scene creation return: licence limit reached."
"SofaContextAPI::loadDefaultPlugin can't load plugins, no sofaPhysicsAPI created!".
How can i fix it?
Thanks in advance for your help

**Comment by epernod** [comment, marked as answer]
Hi @PiccioFra ,

According to the error message, your license has expired. You can request a second one month trial version by re-submitting a request using the [form](https://infinytech3d.com/request-software/) or you can request to upgrade your license to an academic/entreprise SDK. All info are provided [here](https://infinytech3d.com/licenses/)
Best regards,
